docs/.vuepress/config.js

```diff
@@ -1,9 +1,18 @@
+const mdCodesandboxPlugin = require('markdown-it-codesandbox-embed')
```

Comment:
Is this the right one? I only found https://www.npmjs.com/package/markdown-it-codesandbox

Reply:
The issue is that this will create a new CodeSandbox every single time the docs are deployed, which is less than ideal, especially if developers are linking to the CodeSandbox.

Is there a way you can make it so that it doesn't create a new one, and instead just overwrites the existing one if it exists?

Reply:
From CompuIves/codesandbox-client#1311, Ives mention that "the define endpoint creates a sandbox anonymously and reuses the same sandbox if the files are the same". Seems that update functionality is comming but for now I think reusing the same sandbox if files are the same is good.

Reply:
Sure we'll stay with the current solution.